src/DependencyInjection/UnleashEnvVarProcessor.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Unleash\Client\Bundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\EnvVarProcessorInterface;
+use Unleash\Client\Unleash;
+use Closure;
+
+final class UnleashEnvVarProcessor implements EnvVarProcessorInterface
+{
+    public function __construct(private Unleash $client)
+    {
+    }
+
+    public function getEnv(string $prefix, string $name, Closure $getEnv): bool
+    {
+        $value = $this->client->isEnabled($name);
+
+        // Env vars declared from yaml/xml files have string type
+        // 1 : Use unleash value first
+        // 2 : Retrieve the value of declared/default env var of unleash value is false or not retrieved
+        return $value || filter_var($getEnv($name), FILTER_VALIDATE_BOOL);
```

Comment:
Looking at this here, I don't think this is the correct approach... There should be some differentiation between Unleash returning false and the feature not existing. What happens here is that when the feature is evaluated to false a default is used, which IMO is not correct.

What do you think @sighphyre?

Reply:
@RikudouSage Absolutely agree, PHP SDK does support default value as part of the isEnabled call, which is probably a better place for it. That should mean the fallback is only used when the toggle is missing, which is generally the correct flow for an SDK

Side note, if this isn't already using a context provider, I'd consider passing a context as well. Some toggle configurations, like a gradual rollout with a custom stickiness will always resolve (or should, I haven't checked) to false without a context, which is usually not what one wants

Reply:
@sighphyre This uses an instance provided by the Symfony dependency injection, meaning it already has the correct context.

@ybenhssaien Can you use the default value in the call to `isEnabled()`? It should be something like this:

```suggestion
        return $this->client->isEnabled($name, default: filter_var($getEnv($name), FILTER_VALIDATE_BOOL));
```